src/model/document.js

```diff
@@ -148,14 +149,18 @@ export default class Document {
 	/**
 	 * Creates a new top-level root.
 	 *
+	 * **Note**: Creating model root automatically creates corresponding
+	 * {@link module:engine/view/document~Document#roots view root} with the same name. Name of view root
+	 * will be hanged later on when dom root will be {@link module:engine/view/document~Document#attachDomRoot attached}.
```

Comment:
I don't think we need to have such information in the model. The model has nothing to do with view and bing. The information in the editing controller and view is enough.